source/views/components/cells/textfield.js

```diff
@@ -43,6 +44,7 @@ export class CellTextField extends React.Component {
     placeholder: string,
     returnKeyType: 'done' | 'next' | 'default',
     secureTextEntry: boolean,
+    autoCapitalize: string,
```

Comment:
ProTip™: If you have a limited set of possible values, you can make flow check them as an "enum" type:

Doing `autoCapitalize: 'characters' | 'words' | sentences' | 'none'` would allow flow to verify that you're always using one of the allowed values.